CHANGELOG.md

```diff
@@ -11,6 +11,11 @@ Changelog
 * Fabric Path
   * fabricpath_global (@dcheriancisco)
   * fabricpath_topology (@dcheriancisco)
+* Interface
+  * interface_service (@chrisvanheuveln)
+    * `encapsulation_profile_vni`
```

Comment:
Are we anticipating more sub-categories under interface_service?  Is this really needed?